README.md

```diff
@@ -6,4 +6,6 @@ as it can embed text and images to the same latent space.
 
 For more information on the `gpu` usage and `volume` mounting, please refer to the [documentation](https://docs.jina.ai/tutorials/gpu-executor/).
 For more information on CLIP model, please checkout the [blog post](https://openai.com/blog/clip/),
-[paper](https://arxiv.org/abs/2103.00020) and [hugging face documentation](https://huggingface.co/transformers/model_doc/clip.html)
\ No newline at end of file
+[paper](https://arxiv.org/abs/2103.00020) and [hugging face documentation](https://huggingface.co/transformers/model_doc/clip.html)
+
+<!-- version=v0.3 -->
```

Comment:
Don't need this comment any more in a separated repo. Instead, we use tag to control whether publishing a new version or not.